### PR TITLE
New package: txt2man-1.5.6

### PR DESCRIPTION
**Pull request by diogoleal**

### CODE REVIEW

File: srcpkgs/txt2man/template [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'txt2man'
+pkgname=txt2man
+version=1.5.6
+revision=1
+wrksrc=${pkgname}-${pkgname}-${version}
+short_desc="Converts text to man page"
+maintainer="Diogo Leal <diogo@diogoleal.com>"
+license="GPL-2"
+homepage="https://github.com/mvertes/txt2man"
+distfiles="https://github.com/mvertes/txt2man/archive/txt2man-${version}.tar.gz"
```
Comment (Vaelatern) [inline srcpkgs/txt2man/template]: `${homepage}`?

File: srcpkgs/txt2man/template [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+checksum=df9d972c6930576328b779e64aed6d3e0106118e5a4069172f06db290f32586a
+
+do_install() {
+	for docs in *.1; do
+		vman $docs
+    done
```
Comment (Vaelatern) [inline srcpkgs/txt2man/template]: xlint wil complain about your use of spaces to indent.

**Comment by Vaelatern** [comment]
`noarch=yes`

**Comment by diogoleal** [comment]
I made all the changes you suggested

**Pull request merged**